src/index.tsx

```diff
@@ -1,5 +1,6 @@
 import { MantineProvider } from "@mantine/core";
 import "@mantine/core/styles.css";
+import "mantine-datatable/styles.layer.css";
```

Comment:
It looks like `mantine-datatable` is not installed. should this line perhaps be in another PR?